x/ibc/07-tendermint/types/client_state.go

```diff
@@ -30,54 +30,45 @@ type ClientState struct {
 	TrustingPeriod time.Duration `json:"trusting_period" yaml:"trusting_period"`
 	// Duration of the staking unbonding period
 	UnbondingPeriod time.Duration `json:"unbonding_period" yaml:"unbonding_period"`
-	// Latest block height
-	LatestHeight uint64 `json:"latest_height" yaml:"latest_height"`
-	// Latest block time
-	LatestTimestamp time.Time `json:"latest_time" yaml:"latest_time"`
 	// Block height when the client was frozen due to a misbehaviour
 	FrozenHeight uint64 `json:"frozen_height" yaml:"frozen_height"`
+	// Last Header that was stored by client
+	LastHeader Header
```

Comment:
Must store LastHeader so we can pass it into `lite.Verify` as the old header in the update function

Reply:
Since we only ever store 1 header per client at any given time, this is not too bad

Reply:
For now, this is fine - have we checked that we aren't duplicating any data (I guess you did with the timestamp fields)?

Reply:
Checked & seems fine.